lib/Service/SettingsService.php

```diff
@@ -65,6 +65,7 @@ public function getLiveLog(): bool {
 	public function getAppSettings(): array {
 		return [
 			Constants::CONFIG_KEY_SHOWNLEVELS => $this->getShownLevels(),
+			Constants::CONFIG_KEY_LOGLEVEL => $this->config->getSystemValueInt('loglevel', 0),
```

Comment:
```suggestion
			Constants::CONFIG_KEY_LOGLEVEL => $this->config->getSystemValueInt('loglevel', 2),
```

From [1] the default is warning not debug.
[1] https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/logging_configuration.html